src/applications/mhv-landing-page/tests/e2e/pages/LandingPage.js

```diff
@@ -32,30 +32,6 @@ class LandingPage {
   };
 
   visit = props => this.visitPage(props);
-
-  /**
```

Comment:
Note: This had to be removed. The tests were passing in the list of links, but there are different implementations of the toggles that intercept those links. This would become too complex to maintain. Will test the links in other tests.